app/src/main/java/com/example/riden/activities/directionhelpers/DataParser.java

```diff
@@ -0,0 +1,97 @@
+package com.example.riden.activities.directionhelpers;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Vishal on 10/20/2018.
+ */
+
```

Comment:
??

Also can you add a comment here about what are we using this class for. If we want to parese information for a map let's add some comments and also update the name of the class to capture intent.